### ISSUE TITLE
Indent guides showing up in mini-Text Editors

### ISSUE THREAD

**Issue by 50Wliu** [atom/atom#11633]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

### Description

Indent guides are showing up in mini-Text Editors now.

### Steps to Reproduce

1. Open up something that contains a mini-text editor, like find-and-replace, Command Palette, Settings View, etc. etc.
2. Hold down spacebar

**Expected behavior:** Spaces to appear without indent guides

**Actual behavior:** Indent guides appearing

![mini-indent-guides](https://cloud.githubusercontent.com/assets/2766036/14870651/acd670c8-0ca9-11e6-9c12-d469927d63f1.png)

Refs atom/settings-view#767

### Versions

You can get this information from executing `atom --version` and `apm --version` at the command line. Also, please include the OS and what version of the OS you're running.
```
Atom    : 1.9.0-dev-ce4b2fa
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```
```
apm  1.9.3
npm  2.13.3
node 0.10.40
python 2.7.10
git 2.7.0.windows.1
visual studio 2015
```

**Comment by as-cii** [comment]
Thanks for the report, @50Wliu! Are you able to reproduce this problem on #11414? A lot of code has changed on that version, so it might fix this issue too.

**Comment by 50Wliu** [comment]
@as-cii Yup, still happening on 1.9.0-dev-7b2f049.

**Comment by as-cii** [comment]
Ok, so it seems like this bug is present *only* in the display layers branch (ce4b2fa is part of `ns-switch-to-display-layers`). I have fixed this in 1c694df along with other 🐛s, so I'd recommend rebuilding that branch.

Thank you so much @50Wliu for spotting so many issues! 🙇
